Apps/WebClient/src/ClientApp/src/components/navmenu/sidebar.vue

```diff
@@ -463,7 +366,7 @@ export default class SidebarComponent extends Vue {
                                             ]
                                         )
                                     "
-                                    >x</hg-button
+                                    >×</hg-button
```

Comment:
?

Reply:
This replaces the letter x with a multiplication symbol, to match the character that is used for the close button on our alerts.